src/Core/Element.js

```diff
@@ -34,15 +34,15 @@ function elementImmutable (element) {
 
 /**
  * @param {*} content
- * @param {number} index
+ * @param {*} key
  * @return {Element}
  */
-function elementText (content, index) {
+function elementText (content, key) {
```

Comment:
So I pass in `key` (which is `SharedTypeKey`) and the key becomes `SharedTypeKey + SharedTypeKey`? Seems incorrect.

Reply:
Elaborate? It archives the purpose of assigning a key to nodes that cannot get manually assigned keys in author-space. Text nodes returned from a component will consistently have an implicit key of `..` and  text nodes appearing as children would be `.`+index

Reply:
I see now the code is intended to produce `..` as the key. It just seemed confusing while reading the code. Even if the intent is `..` (which could easily be `.#`), passing an explicit `.` (or a variable with a different name) would have made it clearer.

Reply:
`elementText` only supplements the key with a prefix to avoid collision in case the index of a text node is similar to a siblings author provided key.